src/main/java/de/uhd/ifi/se/decision/management/jira/extraction/CodeCommentWithRange.java

```diff
@@ -0,0 +1,21 @@
+package de.uhd.ifi.se.decision.management.jira.extraction;
+
+/** purpose: structure to store comment text
+ * and area it occupies in a source code file
+ */
+public class CodeCommentWithRange {
```

Comment:
[alternative] I suggest to simply call this model class "CodeComment"! [/alternative] 
[pro] It is inherent that a CodeComment has a beginning and an end, it does not need to be made explicit in the class' name. [/pro]